app/lib/shared/configuration.dart

```diff
@@ -138,14 +142,15 @@ class Configuration {
       pubClientAudience: _pubClientAudience,
       pubSiteAudience:
           '621485135717-idb8t8nnguphtu2drfn2u4ig7r56rm6n.apps.googleusercontent.com',
+      adminAudience: 'https://pub.dev',
       credentials: _loadCredentials(),
       blockEmails: true,
       blockRobots: true,
       productionHosts: const ['dartlang-pub-dev.appspot.com'],
       primaryApiUri: Uri.parse('https://dartlang-pub-dev.appspot.com'),
       primarySiteUri: Uri.parse('https://dartlang-pub-dev.appspot.com'),
-      admins: [
-        '111042304059633250784', // isoos
+      adminUserIds: [
+        '6be21685-e61d-465b-ba60-083fb8eb99ad', // isoos
```

Comment:
This is not an oauth2UserId... I think we should make this `adminOauthUserId`, such that it's hardcoded against an external identifier that's easy to lookup.

Reply:
We could also use `email`, I'm a bit torn on that... `email` is easier to read, but `oauthUserId` feel stronger.

Hmm, I think the email for a service account can be reused... so I think `oauthUserId` is better.